api/src/main/java/com/valtech/aem/saas/api/bestbets/BestBetsConsumerService.java

```diff
@@ -0,0 +1,61 @@
+package com.valtech.aem.saas.api.bestbets;
+
+import java.util.List;
+import lombok.NonNull;
+
+/**
+ * Represents a best bets consumer service. Offers the following actions: - add a single best bet - add multiple best
+ * bets - update a best bet - delete a best bet - publish best bets for project - get all best bets
+ */
+public interface BestBetsConsumerService {
+
+  /**
+   * Adds a best bet entry in saas admin.
+   *
+   * @param bestBetPayload object containing details for the best bet.
+   * @throws BestBetsActionFailedException exception thrown when the add action has failed.
+   */
+  void addBestBet(@NonNull BestBetPayload bestBetPayload);
```

Comment:
do we need this method? cant we just say if it is one it is a List with one BestBetPayload object and use the: addBestBets bellow? we will also avoid duplicate code in the implementation

Reply:
The api defines two different endpoints (/api/v3/bestbet and /api/v3/bestbets -> https://decks.valtech.swiss/display/ICSAAS/Best+bets+APIv3).